todo/src/reducers/reducers.js

```diff
@@ -0,0 +1,41 @@
+// Importing from action.js
+import { COMPLETED, ADD_NEW } from "../actions/action";
+
+// Setting the initial state, which is in the store for me to access through todolist.
+const initialState = {
+  todos: [
+    {
+      value: "Example task - Click to complete",
+      completed: false
+    }
+  ]
+};
+
+// building the reducer. which will handle most of the work throughout the app. could also build
+// a reducer combiner to handle this if they get unwieldy
+function reducer(state = initialState, action) {
+  switch (action.type) {
+    case ADD_NEW:
+      const newTodo = {
```

Comment:
this could have just been broken down into the action.payload and returned, you didn't really need to put it into an object first. You could have just did value = [...state, action.payload] then set completed to false. Try not to over complicate the code too much or you'll end up tying too much together and risk writing spaghetti code.